sechub-wrapper-prepare/src/main/java/com/mercedesbenz/sechub/wrapper/prepare/modules/skopeo/SkopeoLocationConverter.java

```diff
@@ -16,7 +20,23 @@ public String convertLocationForLogin(String location) {
         return removeProtocolPrefix(location);
     }
 
-    public String removeProtocolPrefix(String location) {
+    /**
+     * Converts a location to an additional tag. If the location is empty or null, a
+     * default tag is created.
+     *
+     * @param location the location to convert
+     * @return the location without protocol prefix or default tag
+     */
+    public String convertLocationForAdditionalTag(String location) {
+        String withoutProtocol = removeProtocolPrefix(location);
+        if (withoutProtocol.isBlank() || withoutProtocol.isEmpty()) {
+            // we add a random number to prevent conflicts during scans
+            withoutProtocol = DEFAULT_TAG_PREFIX + ":" + new Random().nextInt(1000);
```

Comment:
Hm... Random... 1000...
not really random.

1. I would prefer here something which could not lead to situation that two calls would produce same appendix. For example a UUID, a datetimestamp or just system nano seconds
2. I do not really get the tag purpose here. We use this logic when we download via skopeo from a docker registry. Okay. But why do we need a default flag here, which has a random part at the end? And what exactly do we suppose to download here in this case? Why is the default tag not just "latest" ?!?